src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -94,16 +106,18 @@ public Person toModelType() throws IllegalValueException {
         }
         final Email modelEmail = new Email(email);
 
-        if (address == null) {
-            throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT, Address.class.getSimpleName()));
-        }
-        if (!Address.isValidAddress(address)) {
-            throw new IllegalValueException(Address.MESSAGE_CONSTRAINTS);
-        }
-        final Address modelAddress = new Address(address);
-
-        final Set<Tag> modelTags = new HashSet<>(personTags);
-        return new Person(modelName, modelPhone, modelEmail, modelAddress, modelTags);
+        //        if (address == null) {
+        //            throw new IllegalValueException(
+        //              String.format(MISSING_FIELD_MESSAGE_FORMAT, Address.class.getSimpleName()));
+        //        }
+        //        if (!Address.isValidAddress(address)) {
+        //            throw new IllegalValueException(Address.MESSAGE_CONSTRAINTS);
+        //        }
+        //        final Address modelAddress = new Address(address);
+        //
+        //        final Set<Tag> modelTags = new HashSet<>(personTags);
+        //        return new Person(modelName, modelPhone, modelEmail, modelAddress, modelTags);
+        return new Person(modelName, modelPhone, modelEmail, isDone);
```

Comment:
Do you think there would be a need for a done checker? Just to make sure that "isDone" is either true or false?

Reply:
No need I think. Since whether or not it is true or false does not matter.

Reply:
are there previous checks to ensure that it's a boolean?

Reply:
For user inputs, that check should happen during the write to file. Reading the sample data is cfm boolean cos the sample data is fixed and i wrote it.